packages/frontend-shared/src/gql-components/Auth.vue

```diff
@@ -88,6 +91,7 @@ const props = defineProps<{
   showRetry?: boolean
   showLogout?: boolean
   utmMedium: string
+  showConnectButtonAfterLogin?: boolean
```

Comment:
I'm definitely over-thinking this, but from a prop structure perspective we could make this a bit more flexible (and shorter name) by defining more like `postLoginAction: 'none' | 'connect-project'`. I wouldn't be surprised if we want to support A/B flows through login in the future, so having the more flexible prop structure might come in handy

Reply:
Yeah this can change quite a bit in the next sprint, since we'll be making it work from the launchpad as well. It's possible the component will end up just knowing how to check this on its own and not use a prop at all, and be somewhat combined with CloudConnect stuff, since all logins will then potentially lead to CloudConnect. So a whole bunch of stuff can simplify then I think. I don't know if I'd worry about it in this PR.